galsim/gsobject.py

```diff
@@ -1238,10 +1239,17 @@ def drawImage(self, image=None, nx=None, ny=None, bounds=None, scale=None, wcs=N
         @param surface_ops  A list of operators that can modify the photon array that will be
                             applied in order before accumulating the photons on the sensor.
                             [default: ()]
+        @param sensor       An optional Sensor instance, which will be used to accumulate the
```

Comment:
Josh, I finally got around to doing this.  I added Image methods `bin` and `subsample` which are kind of converse operations to bin pixels together or split them up into subpixels.  And now using fft drawing methods with a sensor defaults to subsampling 3x3.  I also added a bunch of text to the `drawImage` doc string about this and the other new sensor stuff.  Please take a look the recent few commits.